administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -246,20 +246,17 @@
 
 $input      = $this->app->input;
 $layout     = $input->get('layout', '', 'string');
-$aComponent = $input->get('acomponent', '', 'string');
-$aView      = $input->get('aview', '', 'string');
-$extension  = $input->get('extension', '', 'string');
-$rLanguage  = $input->get('referencelanguage', '', 'string') != null ? $input->get('referencelanguage', '', 'string') : '';
+$component  = $input->get('component', '', 'string');
+$rLanguage  = $input->get('referencelanguage', '', 'string');
 ?>
 <button id="toogle-left-panel" class="btn btn-small" 
 		data-show-reference="<?php echo JText::_('COM_ASSOCIATIONS_EDIT_SHOW_REFERENCE'); ?>"
 		data-hide-reference="<?php echo JText::_('COM_ASSOCIATIONS_EDIT_HIDE_REFERENCE'); ?>"><?php echo JText::_('COM_ASSOCIATIONS_EDIT_HIDE_REFERENCE'); ?>
 </button>
 
 <form action="<?php echo JRoute::_(
-			'index.php?option=com_associations&view=association&layout=' . $layout . '&acomponent='
-			. $aComponent . '&aview=' . $aView . '&extension=' . $extension . '&referencelanguage=' . $rLanguage . '&id='
-			. $this->referenceId
+			'index.php?option=com_associations&view=association&layout=' . $layout . '
+			&component=' . $component . '&referencelanguage=' . $rLanguage . '&id=' . $this->referenceId
 		); ?>" method="post" name="adminForm" id="adminForm" class="form-validate" data-associatedview="<?php echo $this->associatedView; ?>">
```

Comment:
$this->component->item instead of $this->associatedView
and remove the $this->associatedView var from the view.